cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +77,22 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
   )
-set_target_properties( igfxcmrt PROPERTIES PREFIX "")
+
+# set the version
+set(CMRT_VERSION_MAJOR 1)
```

Comment:
You probably better use CM runtime version which you already have in cm_def.h for MAJOR and MINOR:
`cmrtlib/agnostic/share/cm_def.h:41:#define CURRENT_CM_VERSION  (CM_7_2)`
You can get it automatically parsing cm_def.h on the fly. Look here for the reference: https://github.com/Intel-Media-SDK/MediaSDK/blob/master/builder/FindFunctions.cmake#L83

This will actually simplify things a lot since customers will be able to deduct soversion from the API version and use dlopen() if needed.

Reply:
yes, it makes sense. Made the change in latest update. Please help review it. Thanks